forgerock-ui-external-libs/pom.xml

```diff
@@ -184,7 +186,7 @@
                             <goal>wget</goal>
                         </goals>
                         <configuration>
-                            <url>https://arshaw.com/xdate/downloads/0.8/xdate.js</url>
+                            <url>https://raw.githubusercontent.com/arshaw/xdate/v0.8/src/xdate.js</url>
```

Comment:
I think this fix is aimed at another problem that xdate.js cannot be obtained from the previous URL.
It should be dealt with as another issue.

Reply:
Thank you for your feedback.
As you said, it is better to divide the issue.
I will cancel this pull request.